src/gauging.jl

```diff
@@ -1,3 +1,9 @@
+function default_cache(ψ::ITensorNetwork)
+  ψψ = norm_network(ψ)
+  return BeliefPropagationCache(ψψ, group(v -> v[1], vertices(ψψ)))
+end
```

Comment:
I don't really like that this has such a general name like `default_cache`, but really is making a cache for a norm network. Maybe call it `default_norm_cache`?